scripts/pg2sphinx_trigger.py

```diff
@@ -122,40 +154,46 @@ def pg_get_tables(sql_query,sql_db):
                     , indexes
                     text
                     );
-                    """)
+                    """
+    )
 
-    c.execute("""
+    c.execute(
+        """
                 create table indexes (
                     id INTEGER PRIMARY KEY
                     , sphinx_index text
                     , index_parent
                     , source text
                     );
-                    """)
+                    """
+    )
     # switch to sqlite3 dictionary mode
     c.row_factory = sqlite3.Row
 
     # Read Sphinx Config
-    with open (SPHINXCONFIG, "r") as myfile:
-        data=myfile.read()
+    with open(SPHINXCONFIG, "r") as myfile:
+        data = myfile.read()
 
     # Parse PG Connection from Sphinx Config
-    CONN_HOST=re.findall('sql_host\s*=\s*(.*)', data)[0]
-    CONN_USER=re.findall('sql_user\s*=\s*(.*)', data)[0]
-    CONN_PWD=re.findall('sql_pass\s*=\s*(.*)', data)[0]
-    CONN_PORT=re.findall('sql_port\s*=\s*(.*)', data)[0]
+    CONN_HOST = re.findall(r'sql_host\s*=\s*(.*)', data)[0]
+    CONN_USER = re.findall(r'sql_user\s*=\s*(.*)', data)[0]
+    CONN_PWD = re.findall(r'sql_pass\s*=\s*(.*)', data)[0]
+    CONN_PORT = re.findall(r'sql_port\s*=\s*(.*)', data)[0]
 
     # parse sphinx config sources and write them to sqlite sources table ...
-    # TODO:
+    # TUDU:
```

Comment:
Probably leave it `TODO` and adapt .pylintrc accordingly in order to "mute" this warning?